canvasapi/account.py

```diff
@@ -616,6 +616,69 @@ def list_enrollment_terms(self, **kwargs):
             **combine_kwargs(**kwargs)
         )
 
+    def list_user_logins(self, **kwargs):
+        """
+        Given a user ID, return that user's logins for the given account.
+
+        :calls: `GET /api/v1/accounts/:account_id/logins \
+        <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.index>`_
+
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.account.Login`
+        """
+        from login import Login
+
+        return PaginatedList(
+            Login,
+            self._requester,
+            'GET',
+            'accounts/%s/logins' % (self.id),
+            **combine_kwargs(**kwargs)
+        )
+
+    def create_user_login(self, user, login, **kwargs):
+        """
+        Create a new login for an existing user in the given account
+
+        :calls: `POST /api/v1/accounts/:account_id/logins \
+        <https://canvas.instructure.com/doc/api/logins.html#method.pseudonyms.create>`_
+
+        :param user: The ID of the user to create the login for
+        :type user: `dict`
+        :param login: The unique ID for the new login
```

Comment:
This param description needs to reflect that `login` is a dictionary, and that a key `unique_id` needs to be passed.